packages/gallery/src/components/gallery/typeValidator/shouldValidate.js

```diff
@@ -0,0 +1,21 @@
+export default function shouldValidate(props, isSSR) {
```

Comment:
you can just have 'shouldValidateTypes' (or reinforce the name) as an argument to the function and not have all the props passed to it.

Reply:
Added to the [next refactoring task ](https://app.asana.com/0/1200247106759123/1200559598586767)